app/src/App.js

```diff
@@ -78,15 +78,10 @@ function createParticle(x, y, type) {
   let destinationX = (Math.random() - 0.5) * 3000;
   let destinationY = (Math.random() - 0.5) * 3000;
   let rotation = Math.random() * 520;
-  // let delay = Math.random() * 200;
 
-  switch (type) {
-    case 'gesture':
-      particle.innerHTML = String.fromCodePoint(0x1F90C)
```

Comment:
**Problem:** Seems like some browser don't support emojis using codepoint.
**Proposal:** Let's try and use an image for the emoji effect

Reply:
Good catch! I've replaced it with an image now